alacritty/src/config/bindings.rs

```diff
@@ -343,6 +347,9 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
         Copy,  +BindingMode::VI; Action::ClearSelection;
         Paste, ~BindingMode::VI; Action::Paste;
         L, ModifiersState::CTRL; Action::ClearLogNotice;
+        T, ModifiersState::LOGO; Action::NewTab;
+        LBracket, ModifiersState::LOGO | ModifiersState::SHIFT; Action::PreviousTab;
```

Comment:
If anyone's curious, this looks like how to change tabs. 

On macOS, the hotkeys are:
- Prev Tab: ⌘ + Shift + [ 
- Next Tab: ⌘ + Shift + ]
- New Tab: ⌘ + T